### DISCUSSION TITLE
Redwoodjs Support

### DISCUSSION THREAD

**Discussion by masterbater** [MichalLytek/typegraphql-prisma#280] [Q&A]
Is there any chance we could use this in redwoodjs project?

**Comment by MichalLytek** [comment]
Have you tried that? What's not working, what are the issues and blockers?

**Reply by masterbater** [reply]
There convention is just to use sdl and I would really love to use this library it more complete than crud generator by redwoodjs , is there any way where typegraphql-prisma emit schema.gql per prisma model?

**Reply by MichalLytek** [reply]
You can try to use `createResolversMap` from `type-graphql` to build typedefs and resolvers:
https://github.com/MichalLytek/type-graphql/blob/master/src/utils/buildTypeDefsAndResolvers.ts#L16
